language/en/interface.json

```diff
@@ -627,6 +627,21 @@
 			"choosePoint": "Please choose a start point!",
 			"tooClose": "You cannot place your start position too close to another player"
 		},
+		"draftOrderMod": {
+			"fair_mode_wait": "%{textColor}Fair mode on, we were waiting for you. ;)",
```

Comment:
why i18n the textcolor bit? if the whole line is meant to be the same color then put it outside

Reply:
removed single color mentions, should I also move it out of the last 3 lines where the are two colors mentioned?
https://github.com/beyond-all-reason/Beyond-All-Reason/pull/2845/commits/b3b70203895ac747c21bac18ac8cea70dad5f753
https://github.com/beyond-all-reason/Beyond-All-Reason/pull/2845/commits/f587dcec2674aa5c2d0b0382c8c5b5bc4d9f8e5f